h2o-r/h2o-package/R/models.R

```diff
@@ -3064,5 +3064,29 @@ h2o.deepfeatures <- function(object, data, layer) {
   h2o.getFrame(dest_key)
 }
 
+setClass("H2OTree", representation(left_children="integer", right_children="integer", thresholds="integer", model_name = "character",tree_number = "integer", tree_class="integer"))
```

Comment:
Please add a R test to demonstrate the functionality.

Reply:
Done.